App.js

```diff
@@ -19,72 +19,52 @@ import {
     AppLogo,
     AppDesc,
     StyledHomeButtonView,
-    StyledHomeText
+    StyledHomeText,
+    StyledLogo
 } from './components/StyledView';
 import { Play } from './pages/Play.js';
 import { HowToPlay } from './pages/HowToPlay.js';
 
 function HomeScreen({ navigation }) {
     return (
         <StyledView>
-            <Image
-                source={require('./assets/genconnect_logo-black.png')}
-                style={{
-                    width: 303,
-                    height: 100,
-                    marginTop: 170
-                }}
-            />
-            <StyledHomeButtonView>
-                <StyledHomeText
-                    style={{ zIndex: 1, left: 10, top: 45 }}
-                    onPress={() => navigation.navigate('Play')}
-                >
-                    {' '}
```

Comment:
What does this {' '} do? If you want to position the text some way, should probably use CSS instead of hard coding an empty space.

Reply:
This is something that's been deleted @sukritsangvong as it's highlighted in red. All changes I've made are indicated with green.

Reply:
oops. my bad. missaw